apps/dapp/components/FeaturedCard.tsx

```diff
@@ -53,11 +64,8 @@ export const FeaturedCard = ({
 
       <div className="flex flex-col gap-1 mt-5 items-left">
         <p className="text-sm">
-          <Dao className="inline mr-2 mb-1 w-4" fill="currentColor" />${TVL} TVL
-        </p>
-        <p className="text-sm text-valid text-success">
-          <Votes className="inline mr-2 mb-1 h-5" fill="currentColor" />
-          {members} {t('info.members')}
+          <Dao className="inline mr-2 mb-1 w-4" fill="currentColor" />$
+          {Number(TVL).toLocaleString()} TVL
```

Comment:
do we want `toFixed(2)` or `toFixed(6)` to set # of decimals or does `toLocaleString()` handle this? do we also want to use `USDC` before tvl as i know we were concerned about showing USDC prices as USD